### PR TITLE
Convert code from `queryselector` to product `array`

### PR DESCRIPTION
**Pull request by Disha-56843**
- product data was get by queryselector which will now get from product array

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -406,122 +406,103 @@ showProducts()
...
-                    return productName2.localeCompare(productName1)
-                })
+            sortIconContainer.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" class='js-z-to-a-sort' height='14px' width:'14px'><!--!Font Awesome Free 6.6.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d="M214.6 41.4c-12.5-12.5-32.8-12.5-45.3 0l-160 160c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L160 141.2 160 448c0 17.7 14.3 32 32 32s32-14.3 32-32l0-306.7L329.4 246.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3l-160-160z"/></svg>`
 
-                productElements.forEach(productElement => productContainer.appendChild(productElement))
+            document.querySelector('.js-z-to-a-sort').addEventListener('click', () => {
+                products.sort((product1, product2) => {
+                    // const productName1 = productElement1.querySelector('.item-name').innerText.toLowerCase()
+                    // const productName2 = productElement2.querySelector('.item-name').innerText.toLowerCase()
```
Comment (utsavsomaiya) [inline script.js:443]: I dont love comment!

Please remove it! Before pushing the code as we discussed you need to review first and then hit me to review!

**Review by utsavsomaiya — CHANGES_REQUESTED** [review]
Okay after this changes all should working as expected. Right?

**Comment by Disha-56843** [comment]
> Okay after this changes all should working as expected. Right?

no I'm still working on it

File: script.js (hunk 1 above)
Comment (Disha-56843) [reply script.js:443]: > I dont love comment!
> 
> Please remove it! Before pushing the code as we discussed you need to review first and then hit me to review!

I want that code for future reference as I'm still working on it

**Comment by utsavsomaiya** [comment]
image

Nothing in git ignore?

File: script.js (hunk 1 above)
Comment (utsavsomaiya) [reply script.js:443]: Yeah but you send me requested review so!

**Comment by Disha-56843** [comment]
> image
> 
> Nothing in git ignore?

yes i made that for vscode folder but now it is not required

**Comment by utsavsomaiya** [comment]
Please delete it!

**Pull request merged**